ONSdigital/dp-dd-frontend-controller
# feature/initial-build-pipeline

### What

Added initial `Dockerfile` and `Jenkinsfile`.

### How to review

* Check the build pipeline for this branch is passing
* Check that the Docker image has been built and pushed to ECR with the correct tag

### Who can review

Anyone but myself.
